### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by danmory**

### CODE REVIEW

File: .github/workflows/app_python.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      - name: Test with pytest
+        run: |
+          pytest
+
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: snyk/actions/setup@master
+      - uses: actions/setup-go@v1
```
Comment (ezio-42) [inline .github/workflows/app_python.yaml]: why we need go here?

**Comment by ezio-42** [comment]
LGTM except https://github.com/innopolis-devops/labs/pull/159#discussion_r974193286

File: .github/workflows/app_python.yaml (hunk 1 above)
Comment (danmory) [reply .github/workflows/app_python.yaml:61]: https://github.com/snyk/actions/tree/master/setup check this

Comment (danmory) [reply .github/workflows/app_python.yaml]: changed to setup-python as said in documentation https://github.com/snyk/actions/tree/master/setup

**Comment by bkmceh** [comment]
Also looks good to me, nice work

**Pull request closed without merging**